src/main/scala/org/hammerlab/guacamole/loci/map/LociMap.scala

```diff
@@ -99,6 +99,19 @@ object LociMap {
   }
 
   /**
+   * Build a map from some LociSets, mapping each set's loci to that sets index in `lociSets`.
+   */
+  def apply(lociSets: Iterable[LociSet]): LociMap[PartitionIndex] = {
```

Comment:
good point, `LociMap` should not know about `PartitionIndex`, but disguising it as an `Int` is not right either; the right solution here is to put this (and a couple other `LociMap` ctors that are specialized to the `LociMap[PartitionIndex]` a.k.a. `LociPartitioning` case into `LociPartitioning`, so I've done that, thanks for flagging this!